From 6a4d71af9920df76a15e427671c90952fddbb411 Mon Sep 17 00:00:00 2001
From: Vincent Untz <vuntz@gnome.org>
Date: Thu, 19 Jan 2012 13:48:40 +0000
Subject: build: Explicitly link to gmodule-2.0

gmodule-2.0 is now gone from the pkg-config files for GTK+ 3 and others.

https://bugzilla.gnome.org/show_bug.cgi?id=667350
---
diff --git a/configure.ac b/configure.ac
index 07f5b7b..aa11046 100644
--- a/configure.ac
+++ b/configure.ac
@@ -55,6 +55,7 @@ GTKSOURCE_REQUIRED_VERSION=3.0
 PKG_CHECK_MODULES(GIGGLE, [
         gio-2.0           >= $GLIB_REQUIRED_VERSION
         glib-2.0          >= $GLIB_REQUIRED_VERSION
+        gmodule-2.0       >= $GLIB_REQUIRED_VERSION
         gthread-2.0       >= $GLIB_REQUIRED_VERSION
         gdk-pixbuf-2.0    >= $GDK_PIXBUF_REQUIRED_VERSION
         gdk-3.0           >= $GTK_REQUIRED_VERSION
--
cgit v0.9.0.2
